packages/jsii-pacmak/lib/targets/java.ts

```diff
@@ -2009,7 +2009,7 @@ class JavaGenerator extends Generator {
     );
 
     props.forEach((prop) =>
-      this.code.line(`private ${prop.fieldJavaType} ${prop.fieldName};`),
+      this.code.line(`protected ${prop.fieldJavaType} ${prop.fieldName};`),
```

Comment:
Package-private (no modifier) is probably a little better since there aren't subclasses involved.

Reply:
Sounds better – My Java is quite rusty 😁